go.mod

```
@@ -1,3 +1,3 @@
 module gocourse05
 
-go 1.21
+go 1.22
```

Пропоную використовувати найновішу на сьогодні версію Go:
```suggestion
go 1.23.0
```

https://go.dev/dl/